### PR TITLE
DM-28362: add cosine_generator and ramp_generator generator functions

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/simactuators/commander_utils.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# This file is part of ts_simactuators.
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline python/lsst/ts/simactuators/commander_utils.py]: Rubin Observatory Telescope and Site System.

File: tests/test_commander_utils.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
+# This file is part of ts_simactuators.
+#
+# Developed for the LSST Data Management System.
```
Comment (couger01) [inline tests/test_commander_utils.py]: Rubin Observatory Telescope and Site System.

**Review by couger01 — CHANGES_REQUESTED** [review]
LGTM except for the license headers.

File: python/lsst/ts/simactuators/commander_utils.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/simactuators/commander_utils.py]: Good suggestion. Done.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**